Contributing-code-to-Oppia.md

```diff
@@ -78,13 +78,15 @@ Welcome! Please make sure to follow the instructions above if you haven't alread
 
 After that, you can choose a good first issue from the [list of good first issues](https://github.com/oppia/oppia/labels/good%20first%20issue). These issues are hand-picked to ensure that you don't run into unexpected roadblocks while working on them, and each of them should have clear instructions for new contributors. If you see one that doesn't, please let us know via [GitHub Discussions](https://github.com/oppia/oppia/discussions) and we'll fix it. For other issues, you might need to be more independent because we might not know how to solve them either.
 
+Please only work on issues that are labelled **"Impact: High"** or **"Impact: Medium"**. (We are not focusing on "Impact: Low" or "Backlog" issues for now; if you submit a fix for those, you will likely be redirected. We also recommend not working on "triage needed" issues since these might get closed or modified during the weekly triage process.)
```

Comment:
I agree with backlog issues and the ones which still needs to be triaged but why not "Impact Low". It might not be required by us at the time being but I think, it could still be a good entry point for newcomers.

Reply:
Because we don't want to spend time reviewing and assessing the correctness of these. We have more than enough issues in the other categories and should focus on them first.

Reply:
Considering what you mentioned re: `Mentors' and reviewers' time is also limited.`, it sounds valid to me. Thanks!